src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -45,6 +45,18 @@ public class ApplicationInsightsLoggerOptions : IOptionsFormatter
         /// </summary>
         public bool EnablePerformanceCountersCollection { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the flag that enables quick pulse collection.
```

Comment:
minor: comment also should say Live Metrics

Reply:
Oh right, my bad